tests/unittests/plotting/test_plot_accessor.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+
+from orion.plotting import PlotAccessor
+from orion.core.utils.tests import create_experiment
+
+
+config = dict(
+    name='experiment-name',
+    space={'x': 'uniform(0, 200)'},
+    metadata={'user': 'test-user',
+              'orion_version': 'XYZ',
+              'VCS': {"type": "git",
+                      "is_dirty": False,
+                      "HEAD_sha": "test",
+                      "active_branch": None,
+                      "diff_sha": "diff"}},
+    version=1,
+    pool_size=1,
+    max_trials=10,
+    working_dir='',
+    algorithms={'random': {'seed': 1}},
+    producer={'strategy': 'NoParallelStrategy'},
+)
+
+
+trial_config = {
+    'experiment': 0,
+    'status': 'new',  # new, reserved, suspended, completed, broken
+    'worker': None,
+    'start_time': None,
+    'end_time': None,
+    'heartbeat': None,
+    'results': [],
+    'params': []
+}
+
+
+def check_regret_plot(plot):
+    assert plot
+    assert "regret" in plot.layout.title.text.lower()
+    assert 2 == len(plot.data)
+
+
+def test_init_require_experiment():
+    with pytest.raises(ValueError):
+        PlotAccessor(None)
+
+
+def test_call_nonexistent_kind():
+    with create_experiment(config, trial_config, ['completed']) as (_, _, experiment):
+        pa = PlotAccessor(experiment)
+        with pytest.raises(KeyError):
```

Comment:
I think ValueError would be more accurate. The error message should be clear for the user also.

Reply:
Good point!